app/models/idea.rb

```diff
@@ -3,7 +3,8 @@ class Idea < ActiveRecord::Base
   include URLNormalizer
   include Publishable
   include Likeable
-  default_scope { order(inspiration: :desc, featured: :desc, comments_count: :desc, created_at: :desc, id: :desc) }
+  default_scope { order(inspiration: :desc) }
```

Comment:
@stedmanblake is there a reason you got rid of the other scopes included here as well?